web/cobrands/fixmystreet.com/layout.scss

```diff
@@ -156,29 +156,69 @@ body.fullwidthpage {
 }
 
 body.frontpage {
+    $fms-logo-width: 200px;
+    $fms-logo-height: 60px;
+    $fms-logo-vertical-margin: 2em;
+    $mysociety-badge-width: 120px;
     #site-logo {
-        margin: 2em 0;
-        width: 300px;
-        height: 60px;
+        margin: $fms-logo-vertical-margin 0;
+        // width: 300px; Size without the mySociety badge
+        width: $fms-logo-width;
+        margin-left: $mysociety-badge-width;
+        position: relative;
+        height: $fms-logo-height;
         background: none;
 
         // Override the width="175" height="35" attributes on the svg element,
         // forcing the SVG contents to stretch to the new, larger, size.
         svg {
-            width: 300px;
-            height: 60px;
+            // width: 300px; Size without the mySociety badge
+            width: $fms-logo-width;
+            float: right;
+            height: $fms-logo-height;
             margin-top: 0; // no need for the vertical centering any more
         }
     }
 
+    #mysociety-badge {
+        top: calc((#{$fms-logo-height} + #{$fms-logo-vertical-margin}) * -1);
```

Comment:
This all seems massively overcomplicated – do we _have_ to do all of this calculation? And why do we need to set a `max-height`?